recipes-extended/onl/onl.inc

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2018 Tobias Jungel <tobias.jungel@bisdn.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+DESCRIPTION = "Open Network Linux Components for Yocto integration"
+HOMEPAGE = "https://www.opennetlinux.org/"
```

Comment:
I think it would make sense to have these to in the .bb file as well, since they will be variant-specific.

Reply:
Done.